IdentityCore/src/MSIDError.h

```diff
@@ -248,6 +248,30 @@ typedef NS_ENUM(NSInteger, MSIDErrorCode)
     // SSO Extension internal error
     MSIDErrorSSOExtensionUnexpectedError           =   -51815,
     
+    // JIT - Link - Timeout while waiting for server confirmation
+    MSIDErrorJITLinkServerConfirmationTimeout      =   -51816,
+    
+    // JIT - Link - Error while waiting for server confirmation
+    MSIDErrorJITLinkServerConfirmationError        =   -51817,
+    
+    // JIT - Link - Error while acquiring intune token
+    MSIDErrorJITLinkAcquireTokenError              =   -51818,
```

Comment:
What is JIT refers to?

Reply:
It's "just in time", the name this feature got.